src/OrchardCore.Modules/OrchardCore.ContentLocalization/Handlers/AutoPartContentLocalizationHandler.cs

```diff
@@ -0,0 +1,21 @@
+using System.Threading.Tasks;
+using OrchardCore.Autoroute.Models;
+using OrchardCore.ContentManagement;
+
+namespace OrchardCore.ContentLocalization.Handlers;
+
+public class AutoroutePartContentLocalizationHandler : IContentLocalizationHandler
```

Comment:
Rename it to `AutoroutePartContentLocalizationHandler` for consistency with `AutorouteContentHandler`